src/SFA.DAS.ApprenticeAccounts.Database/Tables/Apprentice.sql

```diff
@@ -7,6 +7,7 @@
     [DateOfBirth] DATETIME2 NOT NULL,
 	[CreatedOn] DATETIME2 NOT NULL DEFAULT current_timestamp, 
     [TermsOfUseAcceptedOn] DATETIME2 NULL, 
+    [IsPrivateBetaUser] BIT NOT NULL DEFAULT 0, 
```

Comment:
It's probably OK at the scale we are currently at (186k rows), but have you tried applying this change to a database that is a similar size to production?

In very large databases, the time to add the default value to every existing row can be quite significant.

Reply:
Fair point, will aim to go into MO to prove the deploy before go live.

Reply:
Gone into MO, it succeeded minus an issue with the MO restore not pulling across the refactor log tables, so we're all good to go.